modules/react/menu/lib/MenuCard.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+
+import {Card} from '@workday/canvas-kit-react/card';
+import {space, type} from '@workday/canvas-kit-react/tokens';
+import {
+  styled,
+  TransformOrigin,
+  createSubcomponent,
+  StyledType,
+  ExtractProps,
+  createElemPropsHook,
+} from '@workday/canvas-kit-react/common';
+import {getTransformFromPlacement} from '@workday/canvas-kit-react/popup';
+
+import {useMenuModel} from './useMenuModel';
+
+export interface MenuCardProps extends ExtractProps<typeof Card, never> {
+  children?: React.ReactNode;
+}
+
+const StyledCard = styled(Card)<
+  StyledType & {width?: number | string; transformOrigin?: TransformOrigin}
+>(
+  type.levels.subtext.large,
+  {
+    position: 'relative',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  ({transformOrigin}) => {
+    if (transformOrigin == null) {
+      return {};
+    }
+    return {
+      transition: `transform ease-out 150ms`,
+      transformOrigin: `${transformOrigin.vertical} ${transformOrigin.horizontal}`,
+      // Allow overriding of animation in special cases
+      '.wd-no-animation &': {
+        animation: 'none',
+      },
+    };
+  }
+);
+
+export const useMenuCard = createElemPropsHook(useMenuModel)(() => {
+  return {};
+});
+
+export const MenuCard = createSubcomponent('div')({
+  displayName: 'Menu.Card',
+  modelHook: useMenuModel,
+  elemPropsHook: useMenuCard,
```

Comment:
I'm a little confused, can you explain why we need elemPropsHook here? It looks like it's returning an empty object?

Reply:
It is a placeholder in case `useMenuCard` needs to do something more in the future.